crates/cargo-bsp/src/project_model/package_dependency.rs

```diff
@@ -30,7 +31,11 @@ impl PackageDependency {
                 name: dependency.name.clone(),
                 manifest_path: p.manifest_path.clone().into(),
                 optional: dependency.optional,
-                _features: dependency.features.clone(),
+                _features: dependency
+                    .features
+                    .iter()
+                    .map(|f| Feature(f.clone()))
```

Comment:
tego mapowania nie da się wcześniej zrobić?

Reply:
Struktura PackageDependency jest tworzona na podstawie struktury Dependency z cargo właśnie w tym miejscu, dlatego to jest pierwsze miejsce, w którym mapujemy na nasze typy.